### PR TITLE
Google docstring conversion batch 5

### PR DESCRIPTION
**Pull request by michael-berk**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/michael-berk/mlflow/pull/10832?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10832/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10832
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?
Google docstring conversion for store docs.

### How is this PR tested?

- [x] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests

### Does this PR require documentation update?

- [x] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/deployments`: MLflow Deployments client APIs, server, and third-party Deployments integrations
- [x] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/store/_unity_catalog/registry/rest_store.py [hunk 1]
```diff
@@ -368,20 +397,27 @@ def _finalize_model_version(self, name, version):
...
         """
         req_body = message_to_json(FinalizeModelVersionRequest(name=name, version=version))
         return self._call_endpoint(FinalizeModelVersionRequest, req_body).model_version
 
     def _get_temporary_model_version_write_credentials(self, name, version) -> TemporaryCredentials:
         """
         Get temporary credentials for uploading model version files
-        :param name: Registered model name
-        :param version: Model version number
-        :return: mlflow.protos.databricks_uc_registry_messages_pb2.TemporaryCredentials
```
Comment (michael-berk) [inline mlflow/store/_unity_catalog/registry/rest_store.py:383]: TODO

File: mlflow/store/artifact/hdfs_artifact_repo.py [hunk 2]
```diff
@@ -45,10 +47,12 @@ def log_artifact(self, local_file, artifact_path=None):
 
     def log_artifacts(self, local_dir, artifact_path=None):
         """
-            Log artifacts in hdfs.
```
Comment (michael-berk) [inline mlflow/store/artifact/hdfs_artifact_repo.py:48]: Format imporvement

**Comment by michael-berk** [comment]
There is no diff between the local builds of docs html

File: mlflow/store/tracking/dbmodels/models.py [hunk 3]
```diff
@@ -313,7 +316,8 @@ def to_mlflow_entity(self):
         """
         Convert DB model to corresponding MLflow entity.
 
-        :return: :py:class:`mlflow.entities.RunTag`.
+        Returns:
+            mlflow.entities.RunTag: Description of the return value.
```
Comment (serena-ruan) [inline mlflow/store/tracking/dbmodels/models.py:320]: For this file should we keep :py:class:`` wrapper?

File: mlflow/store/tracking/dbmodels/models.py [hunk 4]
```diff
@@ -100,7 +100,8 @@ def to_mlflow_entity(self):
         """
         Convert DB model to corresponding MLflow entity.
 
-        :return: :py:class:`mlflow.entities.Experiment`.
+        Returns:
+            Description of the return value.
```
Comment (serena-ruan) [inline mlflow/store/tracking/dbmodels/models.py:104]: This seems not correct? Also could we keep :py:class: for all return values in this file?

**Review by serena-ruan — APPROVED** [review]
LGTM after updating models.py file

File: mlflow/store/tracking/dbmodels/models.py (hunk 4 above)
Comment (michael-berk) [reply mlflow/store/tracking/dbmodels/models.py]: Great catch!

**Pull request merged**